app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -68,6 +68,7 @@ def reports
       @reports ||= [
         active_users_count_report.active_users_count_emailable_report,
         total_user_count_report.total_user_count_emailable_report,
+        proofing_rate_report.monthly_idv_report_emailable_report,
         proofing_rate_report.proofing_rate_emailable_report,
```

Comment:
ah now I see the memoization issue. make two separate instances here

```suggestion
        proofing_rate_report_30_60_90.monthly_idv_report_emailable_report,
        proofing_rate_report_3_months.proofing_rate_emailable_report,
```

Reply:
I think that an approach like this could also mean there's only one `_as_emailable_report` method too